documentation/release_notes/v4.1.0.md

```diff
@@ -13,7 +13,8 @@
    
    This ticket partially solves this issue introducing `sqw_op` algorithm which allows user to write custom
    operation combining multiple operations performed over filebacked pixels into a single operation. 
-   Algorithm would perform multiple operations, which performance is defined by the speed of IO access, in one go achieving correspondent acceleration.
+   Algorithm would perform multiple operations, which performance is defined by the speed of IO access,
+   in one go achieving correspondent acceleration.
```

Comment:
I have reviewed the available code as best I can. I can find no indication of where this algorithm `sqw_op` is. Please can you make it abundantly clear, either here or in the PR description or in a separate help doc or presentation, which parts of this PR constitute this algorithm and how to use it, so that users and myself can pick it up.

Reply:
see comment below as this message is duplicated at the beginning